### DISCUSSION TITLE
Course platform, impossible to download videos. ¡Help please!

### DISCUSSION THREAD

**Discussion by alejoo93** [aclap-dev/video-downloadhelper#275] [Help and Q&A]
### Your question

I have purchased a course that I only have access to for 1 week, I am trying to download it but it is IMPOSSIBLE. I am attaching screenshots, it is impossible for me to download the video. The plugin detects the videos, but finally it can't download it and I get an error.

Is there anything I can do? 

I attach 2 screenshots, in case it is useful.
![Captura de pantalla 2024-03-08 204713](https://github.com/aclap-dev/video-downloadhelper/assets/114147344/b70fa45b-4490-44db-b751-0ea1c1772ad6)

### Addon & CoApp version and details

![Captura de pantalla 2024-03-08 204525](https://github.com/aclap-dev/video-downloadhelper/assets/114147344/c08954c7-0e5c-489f-822b-fe8d2e772ee7)
![Captura de pantalla 2024-03-08 204713](https://github.com/aclap-dev/video-downloadhelper/assets/114147344/b70fa45b-4490-44db-b751-0ea1c1772ad6)

**Comment by GrampaWildWilly** [comment]
You didn't follow the instructions for supplying the information about your operating system, browser, VDH, & CoApp setup.  You need to provide that information.  Click the Wiki link at the top of this page.  That is a user manual of sorts.  There is a Table of Contents down the right side of that page.  Click the chapter entitled, "I need help."  Read that & this time, follow the instructions.
